src/Components/DialPad/dialPad.component.tsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import Holdable from '../holdable/holdable';
+import {
+  Panel,
+  PanelBody,
+  TeamsThemeContext,
+  Input,
+} from "msteams-ui-components-react";
+import {
+  MSTeamsIcon,
+  MSTeamsIconType,
+  MSTeamsIconWeight
+} from "msteams-ui-icons-react";
+import './dialPad.styles.scss';
+interface Props { }
+interface State {
+  data: string[],
+  buttonLetter: string[],
+  value: string,
+}
+
+class DialPad extends React.Component<Props, State> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      data: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '*', '0', '#'],
+      buttonLetter: ["", "ABC", "DEF", "GHI", "JKL", "MNO", "PQRS", "TUV", "WXYZ", "", "+", ""],
+      value: '',
+    };
+  }
+
+  onHold = (index: number) => {
+    if (this.state.data[index] === '0')
+      this.setState((prevState) => ({
+        value: prevState.value + '+',
+      }))
+  }
+
+  onValueChanged = (event: any) => {
```

Comment:
Why we are not using?
this.setState({
    value: newValue
})

Reply:
the msteams ui library  uses this way to change the state of input library , so I did not want to change it.